### PR TITLE
an explosion pr that doesn't break

### PR DESCRIPTION
**Pull request by AdamElTablawy**
Explosions are less deadly and won't gib anymore. Still very deadly, but they shouldn't instakill, will rarely delimb, and will throw you from the target impact zone. They also don't KO.

This lets me PR actual bomb weapons and grenades.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

:cl:
balance: explosions no longer gib, are less likely to delimb, and throw you from the target impact zone.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -308,35 +308,32 @@
...
 	var/bomb_armor = getarmor(null, "bomb")
 
-	//200 max knockdown for EXPLODE_HEAVY
-	//160 max knockdown for EXPLODE_LIGHT
-
 	switch (severity)
 		if (EXPLODE_DEVASTATE)
 			if(bomb_armor < EXPLODE_GIB_THRESHOLD) //gibs the mob if their bomb armor is lower than EXPLODE_GIB_THRESHOLD
-				for(var/I in contents)
+				/*for(var/I in contents)
```
Comment (buffyuwu) [inline code/modules/mob/living/carbon/human/human_defense.dm:314]: this only damages them now if they have armor thats higher than the gib threshold :) see the else?

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -308,35 +308,32 @@
...
 					var/atom/A = I
 					if(!QDELETED(A))
 						A.ex_act(severity)
-				gib()
+				gib()*/
 				return
 			else
```
Comment (buffyuwu) [inline code/modules/mob/living/carbon/human/human_defense.dm:320]: comment this and the return above it,- hell just comment from the if(bomb_armor and indent the damage code back out of its space

**Review by buffyuwu — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_defense.dm (hunk 2 above)
Comment (AdamElTablawy) [reply code/modules/mob/living/carbon/human/human_defense.dm:320]: noted will fix

**Comment by AdamElTablawy** [comment]
fixed a while ago

**Pull request merged**